packages/svelte/src/compiler/phases/3-transform/client/types.d.ts

```diff
@@ -55,8 +55,8 @@ export interface ComponentClientTransformState extends ClientTransformState {
 	readonly after_update: Statement[];
 	/** The HTML template string */
 	readonly template: {
-		quasi: string[];
-		expressions: Expression[];
+		pushQuasi: (q: string) => void;
```

Comment:
Please don't hate me...since the codebase is in `snake_case` should we name this methods in `snake_case` too? 🫥

Reply:
good catch! done